Python/Product/PythonTools/PythonTools/Environments/VirtualEnv.cs

```diff
@@ -425,5 +425,19 @@ IInterpreterRegistryService service
             }
             return libPath;
         }
+        
+        internal static bool IsPythonVirtualEnv(string prefixPath) {
+            if (string.IsNullOrEmpty(prefixPath)) {
+                return false;
+            }
+            
+            string libPath = FindLibPath(prefixPath);
+            if(libPath == null) {
+                return false;
+            }
+
+            //pyenv.cfg detects python 3 and orig-prefix.txt detects python 2
+            return (File.Exists(Path.Combine(prefixPath, "pyvenv.cfg")) || File.Exists(Path.Combine(libPath, "orig-prefix.txt")));
```

Comment:
Python 3 that has a pyvenv.cfg but no lib folder will return false because it never gets to this line. Maybe that's fine, if lib folder isn't there it's probably not a venv after all.